src/elm/Main.elm

```diff
@@ -1064,29 +847,29 @@ update msg model =
 
         BuildUpdate m ->
             let
-                ( newModel, action ) =
-                    Pages.Build.Update.update model m ( getBuildStepLogs, getBuildStepsLogs ) FocusResult
+                ( newModel, cmd ) =
+                    Pages.Build.Update.update model m
             in
             ( newModel
-            , action
+            , Cmd.map BuildUpdate cmd
```

Comment:
Using Cmd.map here allows us to dispatch Api calls from within Build.Update and then also receive the response in BuildUpdate